test_coordinates

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
Oh this is a mistake. I was trying to check what was going in to test_coordinates using pry but it wasn't working correctly for whatever reason and i guess I made a log of it. This is just garbage, ill delete it.